ng2-components/ng2-alfresco-upload/src/services/upload.service.ts

```diff
@@ -17,132 +17,19 @@
 
 import { EventEmitter, Injectable } from '@angular/core';
 import { Subject } from 'rxjs/Rx';
-import { AlfrescoApiService, AppConfigService } from 'ng2-alfresco-core';
-import { FileUploadEvent, FileUploadCompleteEvent } from '../events/file.event';
-import { FileModel, FileUploadProgress, FileUploadStatus } from '../models/file.model';
-import * as minimatch from 'minimatch';
+import { AlfrescoApiService, BaseUploadService, FileModel } from 'ng2-alfresco-core';
 
 @Injectable()
-export class UploadService {
+export class UploadService extends BaseUploadService {
```

Comment:
Can we just move it entirely to the core lib as 'UploadService'? We had problems in the past when one Injectable extends another Injectable.